packages/firebase_crashlytics/example/pubspec.yaml

```diff
@@ -0,0 +1,21 @@
+name: firebase_crashlytics_example
+description: Demonstrates how to use the firebase_crashlytics plugin.
+
+environment:
+  sdk: ">=2.0.0-dev.68.0 <3.0.0"
+
+dependencies:
+  flutter:
+    sdk: flutter
+
+  cupertino_icons: ^0.1.2
+
+dev_dependencies:
+  flutter_test:
+    sdk: flutter
+
+  firebase_crashlytics:
```

Comment:
Shouldn't this be up in dependencies? (not dev_dependencies?)

Reply:
Done